tests/test_api/test_types/test_user.py

```diff
@@ -14,7 +14,7 @@ class TestUser:
             ["User", " ", "User  "],
             [" ", " ", "   "],
         ],
-    )
-    def test_full_name(self, first: str, last: str, result: bool):
+    )        
+    def test_get_mention_html(self, first: str, last: str, result: bool):
         user = User(id=42, is_bot=False, first_name=first, last_name=last)
-        assert user.full_name == result
+        assert user.mention_html()
```

Comment:
Don't assert string values unless you're checking that something is expected to be non-empty. In your case, you must check if `mention_html` returns exact string. Then try to add arguments and see if it works properly. And don't forget to test `mention_markdown`.